Update doc for service principal certificate authentication (#4756)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/upcoming-breaking-changes.md

- This command group has been deprecated and will be removed in next breaking change release(2.67.0). Use 'az network public-ip ddos-protection' instead.

## profile

### `az login`

> - Passing the service principal certificate with `--password` is deprecated and will be removed in version 2.67.0. Please use `--certificate` instead.

Reviewer: ```suggestion
- The `--password` parameter will not accept service principal certificate starting with version 2.67.0, use the `--certificate` instead.
```

Author: Is the `the` before `--certificate` needed?

Author: Maybe we can align with https://github.com/Azure/azure-cli/issues/30283?
